modin/core/execution/ray/implementations/pandas_on_ray/partitioning/virtual_partition.py

```diff
@@ -504,7 +507,10 @@ class PandasOnRayDataframeRowPartition(PandasOnRayDataframeVirtualPartition):
     axis = 1
 
 
-@ray.remote
+__WORKER_ENV__ = {"env_vars": {"MODIN_LOG_MODE": LogMode.get()}}
```

Comment:
I would suggest to create this in `modin/core/execution/ray/common/utils.py` and import from that everywhere it's needed... or even create a custom decorator in there, like
```python
import ray
import functools

from modin.config import LogMode

__WORKER_ENV__ = {"env_vars": {"MODIN_LOG_MODE": LogMode.get()}}

logged_remote = functools.partial(ray.remote, runtime_env=__WORKER_ENV__)
```
and use it like
```python
from modin.core.execution.ray.common.utils import logged_remote

@logged_remote(num_returns=2)
def func(args):
    # do stuff
    return a, b
```